lib/puppet-strings/yard/handlers/puppet/function_handler.rb

```diff
@@ -27,6 +27,11 @@ class PuppetStrings::Yard::Handlers::Puppet::FunctionHandler < PuppetStrings::Ya
 
     # Mark the class as public if it doesn't already have an api tag
     object.add_tag YARD::Tags::Tag.new(:api, 'public') unless object.has_tag? :api
+
+    # Warn if a summary longer than 140 characters was provided
+    if object.has_tag?(:summary) && object.tag(:summary).text.length > 140
+      log.warn "The length of the summary for function '#{object.name}' exceeds the recommended limit of 140 characters."
```

Comment:
Should we distinguish between Puppet functions and Ruby functions in the warning?